builtin/logical/pki/path_acme_test.go

```diff
@@ -502,6 +502,7 @@ func TestAcmeNonce(t *testing.T) {
 				case "header":
 					req = client.NewRequest(http.MethodHead, "/v1/"+tc.directoryUrl)
 				}
+				//nolint:staticcheck // currently there is no other way to perform this specific request
```

Comment:
If there's no other way, should we just undeprecate it though still suggst `ReadRaw...`?